scripts/await-publish.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const path = require("path");
+const exec = require("child_process").exec;
+
+// Get the current version of the package.
+const versionFilePath = path.join(__dirname, "..", "version.txt");
+const versionFileContents = fs.readFileSync(versionFilePath, "utf-8");
+const [currentVersion, currentIdentifier] = versionFileContents.split("\n");
+
+// Check all of the packages to see if they're updated:
+const nativeDependencies = [
```

Comment:
At some point we should put these all in _one_ place and pull from that.

Reply:
Yes, I'm waiting on a goreleaser feature for that.

Reply:
https://discord.com/channels/890434333251362866/1001177749458784367/1002400366723342446